src/main.rs

```diff
@@ -231,14 +237,9 @@ fn main() {
             let mut tooltip_line = format!(
                 "{} {} {} {}",
                 format_time(hour["time"].as_str().unwrap(), args.ampm),
-                WEATHER_CODES
+                if args.nerd { WEATHER_CODES_NERD } else { WEATHER_CODES }
                     .iter()
-                    .find(|(code, _)| *code
-                        == hour["weatherCode"]
-                            .as_str()
-                            .unwrap()
-                            .parse::<i32>()
-                            .unwrap())
+                    .find(|(code, _)| *code == weather_code.parse::<i32>().unwrap())
```

Comment:
I'm probably a little confused, but how come the `==` was towards `hour["weatherCode"]` before, and now it is towards `weather_code`?

Reply:
That is strange, especially because I didn't think I had touched that part of the code

Reply:
i checked, in one of the first comments you made on the PR you gave me code to replace with, and in that `hour["weatherCode"]` was replaced. Not sure if there's any difference though.